docs/05_best-practices/05_securing_your_contract.md

```diff
@@ -0,0 +1,14 @@
+## Securing your contract
+These are basic recommendations that should be the foundation of securing your smart contract:
+
+1. The master git branch has the `has_auth`, `require_auth`, `require_auth2` and `require_recipient` methods available in the EOSIO library.  They can be found in detail [here](https://eosio.github.io/eosio.cdt/1.6.0-rc1/group__action.html#function-requirerecipient) and implemented [here](https://github.com/EOSIO/eos/blob/3fddb727b8f3615917707281dfd3dd3cc5d3d66d/libraries/chain/apply_context.cpp#L144) (they end up calling the methods implemented in the `apply_context` class).
+
+2. Understand how each of your contracts' actions is impacting the RAM, CPU, and NET consumption, and which account ends up paying for these resources.  You saw earlier how changing one parameter value in the multi-index `emplace` and `modify` methods changed the account that is billed for the RAM and CPU needed to execute that action.
```

Comment:
Instead of "you saw earlier" it should link to the information provided. It appears "best practices" was setup for sequential consumption, but you need to assume that people will be reading these guides out of order.